content/docs/users/_index.md

```diff
@@ -11,7 +11,6 @@ updated = "2022-10-18T16:00:00Z"
 ## Matrix for Instant Messaging
 
 Matrix works a little like email, but instantaneous and secure:
-
```

Comment:
Why?

Reply:
Oh whopsey, that's a rebase that didn't go well 🤦